src/tutorial/testing.md

```diff
@@ -56,7 +56,7 @@ fn check_answer_validity() {
 }
 ```
 
-You can put this snippet of code in pretty much any file
+You can put this snippet of code in pretty much any file in `src/` or `tests/`
```

Comment:
> The current wording might suggest, any arbitrary location for test files.
But, cargo limits it's search to src/ and test/

I feel like this is overly prescriptive and trying to decide how to balance this.  One idea is

```suggestion
You can put this snippet of code in pretty much any source file in your package
```